public/Aframe/loading-screens/scene-1/scene-1.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
```

Comment:
@andstun Can you move these files to the Samples folder? Since we are still discussing the repo structure, having it in the samples folder, for now, will be helpful!

Reply:
That shouldn't be a problem on my end. Did Ahmed give his confirmation? He was the one who outlined how I should structure the folders for my commit. As long as we're good there I can move all the loading screen stuff to samples.

Reply:
Yeah, Jay convinced me. :) Good to go!